scheduler/src/cook/caches.clj

```diff
@@ -16,6 +16,12 @@
       (.expireAfterAccess 2 TimeUnit/HOURS)
       (.build)))
 
+(defn passport-cache [config]
+  "Build a new passport-related cache"
+  (-> (CacheBuilder/newBuilder)
+    (.maximumSize (get-in config [:settings :passport :job-uuid-cache-set-size]))
```

Comment:
nitpick: rename job-uuid-cache-set-size to job-cache-set-size and job-uuid-cache-expiry-time-hours to job-cache-expiry-time-hours

Reply:
Will change